test/sql/utils/testsupport.sql

```diff
@@ -0,0 +1,234 @@
+CREATE SCHEMA IF NOT EXISTS test;
```

Comment:
I wonder if the assert functions I defined previously should go in here as well (not necessarily for this PR).

Reply:
I didn't put them here for now since they are only used in maybe one test, while the functions in this file is basically needed for every test.